app/assets/javascripts/solidus_stripe/stripe-init.js

```diff
@@ -0,0 +1,173 @@
+Spree.stripePaymentMethod = {
```

Comment:
I think this should just be a maniefest, and should require a file with this content. This way it should be easier for stores to override it with a custom version.